wolox-books/src/store.js

```diff
@@ -1,16 +1,27 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import { getBooks } from '@/services/booksService'
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
-
+    books: []
   },
   mutations: {
-
+    setBooks (state, { books }) {
+      state.books = books
+    }
   },
   actions: {
-
+    async getBooks (context) {
+      const response = await getBooks()
+      if (response.ok) {
+        context.commit('setBooks', { books: response.data })
+      }
+    }
+  },
+  getters: {
```

Comment:
This getter is not necessary. You can do this in the component you wanna use the books:
```
computed: {
  ...mapState({
    books: state => state.books
  })
}